From f0e6208f998572a5877e00ed2d09e79a7a432f9c Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Tue, 25 Oct 2011 08:28:44 -0500
Subject: ENGR00160709-1 Move board specific code out of FEC driver

Move the board specific code out of the FEC driver to the platform
layer

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 include/linux/fec.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/linux/fec.h b/include/linux/fec.h
index bcff455..d6a0704 100644
--- a/include/linux/fec.h
+++ b/include/linux/fec.h
@@ -3,7 +3,7 @@
  * Copyright (c) 2009 Orex Computed Radiography
  *   Baruch Siach <baruch@tkos.co.il>
  *
- * Copyright (C) 2010 Freescale Semiconductor, Inc.
+ * Copyright (C) 2010-2011 Freescale Semiconductor, Inc.
  *
  * Header file for the FEC platform data
  *
@@ -17,6 +17,7 @@
 #include <linux/phy.h>
 
 struct fec_platform_data {
+	int (*init) (struct phy_device *);
 	phy_interface_t phy;
 	unsigned char mac[ETH_ALEN];
 };
-- 
1.7.9.5

